public/scripts/prepare-create-assistant.js

```diff
@@ -0,0 +1,84 @@
+import { t } from './i18n.js';
+import { SlashCommandParser } from './slash-commands/SlashCommandParser.js';
+import { getContext } from './st-context.js';
+
+const ASSISTANT_NAME = 'Assistant';
```

Comment:
We already have an exported `neutralCharacterName` constant that you can use.
Should be the same here. So no need to set it again.